src/common/registry/test/key-registry/registry-update.e2e-spec.ts

```diff
@@ -88,8 +91,9 @@ describe('Registry', () => {
 
       await registryService.update(address, blockHash);
       expect(saveOperatorsRegistryMock).toBeCalledTimes(1);
-      // 2 - number of operators
-      expect(saveKeyRegistryMock).toBeCalledTimes(2);
+      //  finalizedUsedSigningKeys == totalSigningKeys
```

Comment:
it would be better to format the left footer of the commentary, it looks like they forgot to delete it